src/pages/blockchain

```diff
@@ -0,0 +1,2 @@
+The blockchain is a distributed database that forms the underlying technological basis for innovations like the bitcoin (cryptocurrency) and Etheruem (smart contracts).
```

Comment:
Add before this line:
```
---
Blockchain
---
```
And also make a directory: `/src/pages/blockchain`, and put there this file with name `index.md`.